frontend/src/components/Profile/ProfilePreview/ProfileMeta/RecruiterProfileMeta.tsx

```diff
@@ -17,7 +18,7 @@ export const RecruiterProfileMeta = React.memo<Props>((props) => {
           className={cn(styles.recruiterMetaItem, typography.smallHeading)}
           key={item.name}
         >
-          {item.text}
+          {item.name !== 'city' ? item.text : <ProfileCity title={item.text} />}
```

Comment:
should we really rely on item name? what if we need an icon for another element? How can we make a code more re-usable?